src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -107,7 +98,7 @@ const EnvironmentPicker = ({
         Boolean(isDialogHiddenProp)
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186236314-044e5a38-ee15-490f-be71-34655028e0ae.png)

Reply:
had commented and marked as "resolved" this on the original PR